hs_core/testing.py

```diff
@@ -110,6 +109,24 @@ def save_files_to_user_zone(self, file_name_to_target_name_dict):
         for file_name, target_name in file_name_to_target_name_dict.iteritems():
             self.irods_storage.saveFile(file_name, target_name)
 
+    def check_file_exist(self, irods_path):
+        """Check whether the input irods_path exist in iRODS using IrodsStorage() object.
+
+        :param irods_path: the iRODS path to check whether it exists or not
+        :return: True if exist, False otherwise.
+        """
+        self.irods_storage = IrodsStorage('federated')
```

Comment:
Is there a reason why we're setting self.irods_storage each time before we use it?